packages/component-base/src/styles/style-props.js

```diff
@@ -6,6 +6,30 @@
 import { css } from 'lit';
 import { addGlobalThemeStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
 
+// NOTE: Base color CSS custom properties are explicitly registered as `<color>`
+// here to avoid performance issues in Aura. Aura overrides these properties with
+// values that use complex, nested relative color functions, and without explicit
+// typing, Chrome 142 was found to render components that use them roughly 40% slower.
+[
+  '--vaadin-text-color',
+  '--vaadin-text-color-disabled',
+  '--vaadin-text-color-secondary',
+  '--vaadin-border-color',
+  '--vaadin-border-color-secondary',
+  '--vaadin-background-color',
+  '--vaadin-background-container',
+  '--vaadin-background-container-strong',
```

Comment:
Noticed that some of these properties are used for shorthand value `background` instead of `backgroud-color`. As a results something like this

```html
<style>
  html {
    --vaadin-background-container-strong: linear-gradient(to right, #ffecd2 0%, #fcb69f 100%);
  }
</style>

<vaadin-text-field label="First name" disabled></vaadin-text-field>
```

would look different after the change:
<img width="478" height="125" alt="Screenshot 2025-11-17 at 9 28 59" src="https://github.com/user-attachments/assets/7325b4aa-4fe8-4a70-89b2-7c1827035329" />

Reply:
I just checked and it appears that the fix doesn't require `--vaadin-background-container` and `--vaadin-background-container-strong` to be registered...

Reply:
`--vaadin-background-container ` and `--vaadin-background-container-strong` are now excluded from the list since they aren't prone to this issue, only `*-color` properties are.